collect_app/src/main/res/values/strings.xml

```diff
@@ -613,6 +613,6 @@
     <string name="hide_old_form_versions_setting_title">Hide old form versions</string>
     <string name="hide_old_form_versions_setting_summary">Only the newest version will appear in Fill Blank Form</string>
     <string name="seconds">Seconds</string>
-    <string name="start_ranking">Start ranking</string>
+    <string name="sort_out_items">Sort out items</string>
```

Comment:
This text doesn't make sense to me. @grzesiek2010 what did you not like about "Start ranking"? Perhaps "Rank items"? @yanokwa?

Reply:
`Start ranking` is ok if there is no answer yet.  I didn't like it was always the same title. 
`Rank items` would be ok.

Reply:
Make sense! Let's go with `Rank items`.

Reply:
Done.

Reply:
🙏